### ISSUE TITLE
Very grateful for your project

### ISSUE THREAD

**Issue by MaaYuu** [skillhacker-code/Kindle-Quran-Website#1]
I just want to say that I'm so grateful for your work. It is so beneficial for me, so good to use this kindle ebook.
JazakAllahu khairan.

By this way, if you made an update to, I'll be appreciated to use it.

**Comment by skillhacker-code** [comment]
Thank you . Happy to know that it is helpful for you . I am still using this version . did you find any errors

**Comment by skillhacker-code** [comment]
yes i tried but kindle was not able to recognise it as words and fetch the translation . 

i converted 0123456789 to zabcdefghi

0 = z
1 = a
2 = b
3 = c
4 = d
5 = e
6 = f
7 = g
8 = h
9 = i


and put "o" between surah number and ayah to seperate it 

that mean surah 1 ayah 7 will be equal to aog
surah 10 ayah 5 will be equal to azoe
